package.json

```diff
@@ -55,15 +55,25 @@
         "@types/through2": "latest",
         "@types/travis-fold": "latest",
         "@types/xml2js": "^0.4.0",
+        "@typescript-eslint/eslint-plugin": "2.0.1-alpha.10",
```

Comment:
Commenting so I don't accidentally merge this until the unified-signatures bug fix is officially released.

Reply:
Updated `typescript-eslint/eslint-plugin` to the latest version `2.1.0`.